crates/fluvio-cluster/src/render.rs

```diff
@@ -1,4 +1,48 @@
+use indicatif::ProgressBar;
+
 pub trait ProgressRenderedText {
     /// Rendered text of the last finished step in the progress
     fn msg(&self) -> String;
 }
+
+#[derive(Debug)]
+pub enum ProgressRenderer {
+    /// Render the progress using eprintln macro
+    Std,
+    /// Render the progress using Indicatiff
+    Indicatiff(ProgressBar),
+}
+
+impl ProgressRenderer {
+    pub fn println(&self, msg: String) {
```

Comment:
if this is change to: `pub fn println(&self, msg: &str)` then don't need `into()`